.travis.yml

```diff
@@ -20,5 +20,5 @@ before_script:
 script:
   - npm test
 after_success: 
-  - npm run coveralls
+  - npm run coverage
   - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
```

Comment:
EOF missing. There should be and End Of File line added to every file

Reply:
This is a rule defined in the Airbnb rules

Reply:
Please, can I go ahead and merge this branch?